File: blook_app_flutter/lib/models/edit_user_dto.dart
class EditUserDto {
  EditUserDto({
    required this.name,
    required this.lastname,
    this.email,
  });
  late final String name;
  late final String lastname;
  late String? email;

  Map<String, dynamic> toJson() {
    final _data = <String, dynamic>{};
    _data['name'] = name;
    _data['lastname'] = lastname;
    _data['email'] = email;
    return _data;
  }
}

File: blook/src/main/java/com/salesianostriana/blook/dtos/EditUserDto.java
package com.salesianostriana.blook.dtos;

import com.salesianostriana.blook.validation.anotations.EditEmail;
import lombok.*;
import javax.validation.constraints.Email;
import javax.validation.constraints.NotBlank;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
@EditEmail(nickField = "nick",
        emailField = "email",
        message = "{user.unique.email}")
public class EditUserDto {

    private String nick;
    private String name;
    private String lastname;
    @NotBlank(message = "{user.email.blank}")
    @Email(message = "{user.email.email}")
    private String email;
}